planner/core/explain.go

```diff
@@ -472,7 +472,7 @@ func (p *PhysicalMergeJoin) ExplainNormalizedInfo() string {
 func (p *PhysicalTopN) ExplainInfo() string {
 	buffer := bytes.NewBufferString("")
 	buffer = explainByItems(buffer, p.ByItems)
-	fmt.Fprintf(buffer, ", offset:%v, count:%v", p.Offset, p.Count)
+	fmt.Fprintf(buffer, ", offset:%v, estRows:%v", p.Offset, p.Count)
```

Comment:
ditto

Reply:
So the strings of `ExplainInfo()`:
- In `PhysicalIndexLookUpReader`, `PhysicalLimit` or `PhysicalTopN`, it should be `count`.
- In `LogicalTopN`, `LogicalLimit`, it should be `estRows`.

Is this right?